Spec/Auth.swift

```diff
@@ -1106,6 +1106,36 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
+                tokenParams.clientId = "newClientId"
+                tokenParams.ttl = 2000
+                tokenParams.capability = "{ \"test:*\":[\"test\"] }"
+                
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.createTokenRequest(tokenParams, options: authOptions) { tokenRequest, error in
```

Comment:
Ok, this might be an "iOS" thing then that I have misunderstood.  I have never written iOS code so just looking at these PRS with a view to a) does it achieve the spec, b) does it follow the API.  @tcard can you comment on this and confirm you're happy to leave as is i.e. I was wrong :)

Thanks @EvgenyKarkan for the explanation